tasks-by-users/Naufil-Shemle/02-naufil-css-basics/02-basic-bike-website/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Bikes!</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div id="wrapper">
+      <nav id="main-navigation">
+        <ul>
+          <li><a href="#" class="current">Bikes</a></li>
+
+          <li><a href="#">Best Buy</a></li>
+          <li><a href="#">Offers</a></li>
+          <li><a href="#">Accessories</a></li>
+          <li><a href="#">Contect</a></li>
+          <li><a href="#">About</a></li>
+        </ul>
+      </nav>
+
+      <div id="lead-banner">
+        <img src="images/lead-banner.jpg" alt="Bikes" />
+      </div>
+
+      <section id="shop">
+        <img
+          class="product-image"
+          src="images/bike.jpg"
+          alt="RetroSyperb image"
+        />
+        <h2 class="product-name">Bicycle RetroSyperb Vii #1</h2>
+
+        <span class="price">299.99$</span>
+        <a href="#" class="add-to-bag">Add to shoping basket</a>
+        <img class="rating" src="images/stars.png" alt="star-rating" />
+      </section>
+
+      <div id="footer-banner">
```

Comment:
you can use `footer` here @Naufil-Shemle

Reply:
I just followed the tutorial, in that he used footer-banner